src/keys.ts

```diff
@@ -342,22 +341,19 @@ export function validateExtendedPublicKeyForNetwork(
 export function validateExtendedPublicKey(
   xpubString: string,
   network: BitcoinNetwork
-): string {
+): string | void {
```

Comment:
Yeah, I thought so too but my linter was highlight this return typing so I assumed it was considering an empty string as void because it's falsey i think? I'll try and put it back and double check the error.

Reply:
Oh maybe!! 🤷🏻 
I could be wrong about that.

Reply:
nvm looks like the void was unnecessary afterall

Reply:
Yeah, really strange. Thanks for catching.